tests/test_integration.py

```diff
@@ -455,6 +456,21 @@ def test_info(self):
         self.vm.shutdown()
         host_info(VM_HOSTNAME)
 
+    @patch('igvm.vm.VM.vm_performance_value', return_value=5.0)
+    def test_igvm_migration_log(self, mock_vm_performance_value):
+        regex = r'\A([0-9]{10}\s[-|+][0-9]{1,3})+\Z'
+
+        for hv in self.hvs:
+            if hv.dataset_obj['hostname'] == self.vm.dataset_obj['hypervisor']:
+                hv.dataset_obj['igvm_migration_log'] = ''
+                hv.dataset_obj.commit()
+                self.assertEqual(str(hv.dataset_obj['igvm_migration_log']), '')
```

Comment:
Evaluating the boolean representation directly instead of the string representation is possible.
```
>>> h['igvm_migration_log']
MultiAttr()
>>> bool(h['igvm_migration_log'])
False
>>> h['igvm_migration_log'].add('1591169717 +5')
>>> bool(h['igvm_migration_log'])
True
```

Reply:
Thats right. I'm testing the bool now.